packages/cmd/src/syntax/SyntaxValidator.ts

```diff
@@ -803,7 +859,7 @@ export class SyntaxValidator {
                     "The value must be a number",
```

Comment:
Fun fact, this `validateNumeric` allows for things like `"1e-2"` which is technically equal to 0.01 :smile:
I'm grateful for simple implementations :+1:

(I'm putting the comment in here because GitHub wouldn't let me place on the `isNaN` line)